### PR TITLE
update email notif message

### PR DESCRIPTION
**Pull request by annalisev**
Add emojis to make the emails more engaging.

### CODE REVIEW

File: pipeline/notification/alert.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 STRING_1 = 'Hey there!\n\nJust to let you know there are new games available to play in the '
```
Comment (hhm970) [inline pipeline/notification/alert.py:13]: Good use of global variables

**Review by hhm970 — APPROVED** [review]
Code is coherent; the game emoji in the message really makes it more engaging XD

**Review by shindym — APPROVED** [review]
Looks good !

**Review by Eibironke — APPROVED** [review]
looks good to me

**Pull request merged**